Multisig/UI/UI Library/Overlays/Snackbar/SnackbarCenter.swift

```diff
@@ -48,7 +48,9 @@ class SnackbarCenter: ObservableObject {
     }
 
     func resetBottomPadding() {
-        bottomPaddingStack.removeLast()
+        if !bottomPaddingStack.isEmpty {
```

Comment:
this fixes a crush on iOS 14

Reply:
![dog-gif](https://user-images.githubusercontent.com/284056/93868740-6e211580-fcba-11ea-9321-4a44ad2b3fb8.gif)